app/config/environment.d.ts

```diff
@@ -3,8 +3,8 @@
  *    import config from 'my-app/config/environment'
  */
 declare const config: {
-  VO_HEADER_WIDGET_URL: any;
-  VO_FOOTER_WIDGET_URL: any;
+  VO_HEADER_WIDGET_URL: string;
+  VO_FOOTER_WIDGET_URL: string;
```

Comment:
Are these still needed? I think we're just using the Appuniversum based header and footer?

Reply:
I think you're correct!